include/dwg.h

```diff
@@ -8064,6 +8064,17 @@ enum {
   FLAG_R11_HAS_ATTRIBS = 128,
 };
 
+enum {
```

Comment:
Those test generation flags are actually for all versions, not just R11

Reply:
I understand. First version :-)